api/v2/openapi.yaml

```diff
@@ -80,9 +80,14 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/Location'
+                type: object
+                properties:
+                  coordinates:
+                    $ref: '#components/schemas/Coordinates'
+                  locations:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Location'
```

Comment:
Ich fände es schöner, wenn das `LocationSearchResult` unten in die `schemas` verschoben wird, dann wird es auf dauer übersichtlicher.